com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -330,50 +330,26 @@ private static void copyBundleResources(Project project, File outDir) throws Com
     /**
     * Copy Android resources per package
     */
-    private static File copyResources(Project project, Platform platform, File outDir, BundleHelper helper, ICanceled canceled) throws IOException, CompileExceptionError {
+    private static File copyLocalResources(Project project, File outDir, BundleHelper helper, ICanceled canceled) throws IOException, CompileExceptionError {
         File androidResDir = createDir(outDir, "res");
+        androidResDir.deleteOnExit();
 
-        // Native extension build
-        // Include all Android resources received from extender server
-        File packagesDir = new File(project.getRootDirectory(), "build/"+platform.getExtenderPair()+"/packages");
-        File packagesList = new File(packagesDir, "packages.txt");
-        if(packagesList.exists()) {
-            // create a list of resource directories, per package
-            List<File> directories = new ArrayList<>();
-            List<String> allLines = Files.readAllLines(packagesList.toPath());
-            for (String line : allLines) {
-                File resDir = new File(packagesDir, line);
-                if (resDir.isDirectory()) {
-                    directories.add(resDir);
-                }
-            }
-
-            // copy all resources per package
-            // packages/com.foo.bar/res/* -> res/com.foo.bar/*
-            for (File src : directories) {
-                Path srcPath = Path.of(src.getAbsolutePath());
-                String packageName = srcPath.getName(srcPath.getNameCount() - 2).toString();
-                File dest = createDir(androidResDir, packageName);
-                for(File f : src.listFiles(File::isDirectory)) {
-                    FileUtils.copyDirectoryToDirectory(f, dest);
-                    BundleHelper.throwIfCanceled(canceled);
-                }
-            }
+        if (ExtenderUtil.hasNativeExtensions(project)) {
+            log("Trying to copy resources during an extension build - ignoring!");
```

Comment:
This check should probably be moved outside of the function.
I think a function named "copy" should copy (as opposed to sometimes _not_ copy)

Reply:
Good point. And the function will not get called in an extension build either so the check is not needed.